### ISSUE TITLE
No SF-icons viable

### ISSUE THREAD

**Issue by iAmRenzo** [menahishayan/HomeAssistant-Cupertino-Icons#17]
I have no idea how or what, but on two instances of Home Assistant I cannot get cupertino icons to work.
![image](https://github.com/user-attachments/assets/c1773a7a-8ed7-4d88-8a48-994f60826f60)

I did all HAC steps, rebooted, refreshed... nothing. Is this project perhaps abandoned?

**Comment by FezVrasta** [comment]
Same issue here

**Comment by iAmRenzo** [comment]
I think the developer abandoned this project...

**Comment by lnlyssg** [comment]
After a _lot_ of help from ChatGPT I've managed to get it working again, my fork is at https://github.com/lnlyssg/HomeAssistant-Cupertino-Icons

**Comment by menahishayan** [comment]
Hey everyone, sorry there's a lot going on right now so I haven't been able to find much time to be active here, but thanks @lnlyssg for this fork! It would be great if everyone can benefit from this fix. Would you like to create a PR?

**Comment by lnlyssg** [comment]
Yup will do. I also pulled in another user's fork with SF Symbols 6 as well as fixing the future deprecations. I'll try get a PR submitted at some point this weekend.

**Comment by menahishayan** [comment]
Sorry for the delayed response. #19 has been merged. Would anyone be able to confirm if this fixes the issue?
